core/field_dropdown.js

```diff
@@ -63,6 +63,10 @@ Blockly.FieldDropdown = function(menuGenerator, opt_validator) {
   this.trimOptions_();
   var firstTuple = this.getOptions()[0];
 
+  // Call parent's constructor.
+  Blockly.FieldDropdown.superClass_.constructor.call(this, firstTuple[1],
```

Comment:
Q: Do you think moving this super to the top of the constructor needs looking into (after this PR)?

Reply:
Yep, and there's an obvious place for it to go (hint: initValue).